### PR TITLE
missing time zones

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

File: data/gaTable.yaml [hunk 1]
```diff
@@ -515,7 +515,7 @@
   webPropertyId: UA-25350863-1
   internalWebPropertyId: '49291662'
   viewName: ''
-  timezone: ''
+  timezone: 'America/Denver'
```
Comment (ldecicco-USGS) [inline data/gaTable.yaml]: @wdwatkins could you go here:
https://github.com/USGS-VIZLAB/internal-analytics/blob/master/scripts/visualize/timeDayUse.R#L67

and get Denver (and any other non-central times that need correction) in there?

Comment (wdwatkins) [inline data/gaTable.yaml]: added now

**Comment by ldecicco-USGS** [comment]
I got an error on that `recode` line, but if I shuffle it around just a little to this (that is...turn all the hours to numeric first, then do the recode):
```
  viz.data <- left_join(viz.data, projTZ, by = 'viewID') %>%
    mutate(hour = as.numeric(hour),
           hour = recode(timezone,
                      `America/New_York` = hour,
                      `America/Chicago` = hour + 1,
                      `America/Denver` = hour + 2,
                      `America/Los_Angeles` = hour + 3))

  hourSum <- viz.data %>%
    group_by(hour) %>%
    summarise(Sessions = n())
```

then it works (plus, I removed a as.numeric(as.character()) set that didn't need to be there)

**Comment by wdwatkins** [comment]
This should be good to go now

**Pull request merged**